streamer/src/streamer.rs

```diff
@@ -147,7 +148,9 @@ fn recv_loop(
                     if len == PACKETS_PER_BATCH {
                         full_packet_batches_count.fetch_add(1, Ordering::Relaxed);
                     }
-
+                    packet_batch
+                        .iter_mut()
+                        .for_each(|p| p.meta_mut().set_from_staked_node(is_staked_service));
```

Comment:
https://rust-lang.github.io/rust-clippy/master/index.html#/needless_for_each

Reply:
clippy didn't flag it, see above comment about style.